src/fpm_command_line.f90

```diff
@@ -52,10 +53,28 @@ module fpm_command_line
 
 type, abstract :: fpm_cmd_settings
     character(len=:), allocatable :: working_dir
-    logical                      :: verbose=.true.
+    logical                       :: verbose=.true.
 end type
 
 integer,parameter :: ibug=4096
+
+type, extends(fpm_cmd_settings)   :: fpm_global_settings
+    !> Path to the global config file excluding the file name.
+    character(len=:), allocatable :: path_to_folder
+    !> Name of the global config file. The default is `config.toml`.
+    character(len=:), allocatable :: file_name
+    type(fpm_registry_settings), allocatable :: registry_settings
+contains
+    procedure :: full_path    
+end type
+
+type, extends(fpm_cmd_settings)   :: fpm_registry_settings
```

Comment:
I'm a bit confused here, the registry setting is not really a command line setting. It is also handled completely different from any other setting defined here.

I wonder whether it makes sense to actually have the settings here or move them into a new module to keep them logically separated.

Reply:
We could at some point support entering the registry path via the CLI. Then it would behave similarly to the others. Is that wanted? If no, then yes, this should go somewhere else.

Reply:
No, I think I understand now. How about `fpm_global_settings`, should it also be moved to somewhere else?